src/components/Quote.vue

```diff
@@ -41,13 +41,14 @@ components.
 					disable-menu />
 				{{ getDisplayName }}
 			</div>
-			<div v-if="isFileShareMessage"
-				class="quote__main__text">
-				<NcRichText :text="message"
-					:arguments="richParameters"
-					:autolink="true" />
-			</div>
-			<blockquote v-else
+			<!-- file preview-->
+			<NcRichText v-if="isFileShareMessage"
+				text="{file}"
+				class="quote__file__preview"
+				:arguments="richParameters"
+				:autolink="true" />
```

Comment:
Do we need `autolink`, if it's file only?

Reply:
forgot to remove it

Reply:
and captions with links?